options/locale/locale_en-US.ini

```diff
@@ -1062,6 +1062,7 @@ pulls.data_broken = This pull request is broken due to missing fork information.
 pulls.files_conflicted = This pull request has changes conflicting with the target branch.
 pulls.is_checking = "Merge conflict checking is in progress. Try again in few moments."
 pulls.required_status_check_failed = Some required checks were not successful.
+pulls.required_status_check_missing = Some required checks are missing.
```

Comment:
How will this work with Crowdin now?

Reply:
maybe should translate it manually ...  :(

Reply:
it will work just fine, crowdin checks only master branch. In this case this text will be in English in all languages